src/types/onyx/RecentlyUsedTags.ts

```diff
@@ -0,0 +1,3 @@
+type RecentlyUsedTags = string[];
```

Comment:
This will be an object of string[] since there can be multiple tag lists

Example data:
```
"Location": [
            "New York",
            "Portland",
            "San Francisco",
            "London",
            "Ironwood"
        ],
        // ... For multi-tags, additional tag lists
```